arcsi/static/docs.json

```diff
@@ -0,0 +1,960 @@
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
+            },
+            "language": {
+              "type": "string"
+            },
+            "cover_image_url": {
+              "type": "string"
+            },
+            "playlist_name": {
+              "type": "string"
+            },
+            "frequency": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "week": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "day": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "start": {
+              "type": "string"
+            },
+            "end": {
+              "type": "string"
+            },
+            "archive_lahmastore": {
+              "type": "boolean"
+            },
+            "archive_lahmastore_base_url": {
+              "type": "string"
+            },
+            "archive_mixcloud": {
+              "type": "boolean"
+            },
+            "archive_mixcloud_base_url": {
+              "type": "string"
+            },
+            "items": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/components/schemas/item"
+              }
+            },
+            "users": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/components/schemas/user"
+              }
+            }
+          }
+        },
+        "ShowRequests": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/components/schemas/ShowRequest"
+          }
+        },
+        "ShowRequestPostBody": {
+          "type": "object",
+          "properties": {
+            "active": {
+              "type": "boolean"
+            },
+            "name": {
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
+            },
+            "language": {
+              "type": "string"
+            },
+            "playlist_name": {
+              "type": "string"
+            },
+            "frequency": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "week": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "day": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "start": {
+              "type": "string"
+            },
+            "end": {
+              "type": "string"
+            },
+            "archive_lahmastore": {
+              "type": "boolean"
+            },
+            "archive_mixcloud": {
+              "type": "boolean"
+            },
+            "users": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "user_name": {
```

Comment:
These are actually POST'ed to our API from the form, maybe put something of an example to these two? 

F.e.: `user_name` might be `Arcsi` and `user_email` `arcsi[at]lahmacun[dot]hu` ?

Reply:
I added every kind of examples to the PostRequestBodies